inference-engine/src/vpu/graph_transformer/src/stages/broadcast.cpp

```diff
@@ -104,18 +103,22 @@ void FrontEnd::parseBroadcast(
         const DataVector& outputs) const {
     VPU_THROW_UNLESS(layer != nullptr,
                      "parseBroadcast expects valid CNNLayerPtr, got nullptr");
-
     VPU_THROW_UNLESS(outputs.size() == 1,
                      "{} layer with name {} must have only 1 output, actually provided {} outputs",
                      layer->type, layer->name, outputs.size());
     const auto output = outputs[0];
-
     const auto modeString = layer->GetParamAsString("mode", "numpy");
-    if (modeString == "numpy") {
+    const std::map<std::string, BroadcastMode> modeFromString = {
+        {"numpy", BroadcastMode::NUMPY},
+        {"explicit", BroadcastMode::EXPLICIT},
+        {"bidirectionsl", BroadcastMode::BIDIRECTIONAL}
+    };
+    const auto mode = modeFromString.at(modeString);
```

Comment:
`at` throws an exception in the absence of `modeString` in map. You must check it by yourself (e.g. using `find` and comparing with `modeFromString.end()`)